templates/_bundle_builder/Dockerfile

```diff
@@ -19,28 +19,28 @@ RUN apt-get update && \
 #
 
 RUN apt-get update && \
-    \
-    DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install --yes libass-dev libfaac-dev libmp3lame-dev libopus-dev libsdl2-dev libtheora-dev libvorbis-dev libvpx-dev libx264-dev libx265-dev yasm
-
-RUN curl -L -o /build/ffmpeg.tar.gz https://ffmpeg.org/releases/ffmpeg-7.1.tar.gz && \
-    \
-    cd /build && \
-    \
-    tar -xvzf ffmpeg.tar.gz && \
-    \
-    cd ffmpeg-* && \
-    \
-    mkdir build && \
-    \
-    cd build && \
-    \
-    ../configure --enable-gpl --enable-libass --enable-libfreetype --enable-libmp3lame --enable-libopus --enable-libtheora --enable-libvorbis --enable-libvpx --enable-libx264 --enable-libx265 --enable-nonfree --enable-sdl2 --enable-shared --enable-version3 && \
-    \
-    make -j$(nproc) && \
-    \
-    make install && \
-    \
-    rm -rf /build/*
+	\
+	DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install --yes libass-dev libfaac-dev libmp3lame-dev libopus-dev libsdl2-dev libtheora-dev libvorbis-dev libvpx-dev libx264-dev libx265-dev yasm
+
+RUN curl --location --output /build/ffmpeg.tar.gz https://ffmpeg.org/releases/ffmpeg-7.1.tar.gz && \
+	\
+	cd /build && \
+	\
+	tar --extract --verbose --gzip --file ffmpeg.tar.gz && \
```

Comment:
@Akos-Kreutz, can we sort alphabetically?

Reply:
Sadly no,
 tar arguments can only be rearranged to alphabetical order to a certain point as --file needs to be followed by the filename thus needs to be at the end.